### PR TITLE
dms: detect and forward scope

### PR DESCRIPTION
**Pull request by tarcisiozf**
* refactor: centralize ws actions for order submitting
* add scope to algo/atomic order meta
* add scope to auth actions

### CODE REVIEW

File: src/redux/actions/ws.js [hunk 1]
```diff
@@ -9,8 +10,13 @@ const send = payload => ({
     : JSON.stringify(payload),
 })
 
+const getScope = () => {
```
Comment (dmytroshch) [inline src/redux/actions/ws.js:13]: It shouldn`t be in actions

Comment (tarcisiozf) [reply src/redux/actions/ws.js:13]: where do you think it would be a better place? I'm not so familiar with the code organization on this repo

Comment (tarcisiozf) [reply src/redux/actions/ws.js:13]: maybe in the [config](https://github.com/tarcisiozf/bfx-hf-ui-core/blob/external-dms/src/redux/config.js), where the isElectronApp constant is being set?

Comment (tarcisiozf) [reply src/redux/actions/ws.js]: updated: d443173290aa86a2c563e0475e03a998e25552e7

**Review by bhoomij — APPROVED** [review]

**Pull request merged**